app/views/layouts/application.html.erb

```diff
@@ -59,21 +59,6 @@
     navigation_items: navigation_items,
   } %>
 
-  <nav id="mobile-navigation" class="accounts-mobile-menu js-mobile-menu" aria-hidden="true" role="navigation">
-    <ul class="accounts-mobile-menu__list">
-      <% navigation_items.each do |item| %>
-        <li>
-          <% if item[:sub_menu].present? %>
```

Comment:
Could you also remove the `sub_menu` bit from https://github.com/alphagov/govuk-account-manager-prototype/blob/main/app/helpers/application_helper.rb#L15 ?

Reply:
Good spot, thanks, removed.